src/tools/rbd/action/Kernel.cc

```diff
@@ -500,17 +512,11 @@ int execute_map(const po::variables_map &vm,
     return r;
   }
 
-  // parse default options first so they can be overwritten by cli options
-  r = parse_map_options(
-      g_conf().get_val<std::string>("rbd_default_map_options"));
-  if (r < 0) {
-    std::cerr << "rbd: couldn't parse default map options" << std::endl;
-    return r;
-  }
+  MapOptions map_options;
 
```

Comment:
Nit: There is no empty line here in master -- removing it would make the diff cleaner.

Reply:
Thanks. Updated